### PR TITLE
feat(frontend): redesign signup page

### PR DESCRIPTION
**Pull request by zaxutic**
signup page form

### CODE REVIEW

File: frontend/src/components/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  form-input w-96 rounded-md
+  border-gray-300 shadow-sm
+  transition hocus:border-blue-300 focus:(ring ring-blue-200/50)
+
+  invalid:(
+    border-red-300 text-red-600 ring-red-200/50
+    hover:(border-red-400)
+  )
+`;
+const Label = tw.label`flex flex-col`;
```
Comment (Varun-Sethu) [inline frontend/src/components/Input.tsx:15]: very cool

File: frontend/src/pages/signup/index.tsx [hunk 2]
```diff
@@ -28,67 +31,81 @@ const Signup = () => {
...
-    <BackgroundWrapper>
-      <StyledForm
+    <Container tw="flex-none justify-center gap-2 text-black">
+      <header tw="text-center">
+        <img tw="h-12 mx-auto filter drop-shadow" src={chaosImg} alt="Chaos" />
+        <div tw="my-4">
+          <h1 tw="text-4xl">Create an account</h1>
+          <p tw="text-gray-600">To start simplifying recruitment</p>
+        </div>
+      </header>
```
Comment (Varun-Sethu) [inline frontend/src/pages/signup/index.tsx:41]: ebic

File: frontend/src/pages/signup/index.tsx [hunk 3]
```diff
@@ -28,67 +31,81 @@ const Signup = () => {
...
+            <Input.LabelText required>zID (zXXXXXXX)</Input.LabelText>
+            <Input
+              required
+              value={formData.zid}
+              onChange={(e) =>
+                setFormData({ ...formData, zid: e.target.value })
+              }
+              pattern="z\d{7}"
+            />
+          </Input.Label>
```
Comment (Varun-Sethu) [inline frontend/src/pages/signup/index.tsx:73]: Good JOB!!

**Review by Varun-Sethu — COMMENTED** [review]
LGTM!!! 🚀

File: frontend/src/components/Input.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+type LabelTextProps = {
+  /**
+   * Whether this form control is required.
+   * If this is true, a red asterisk is shown next to the label text
+   */
+  required?: true;
```
Comment (zaxutic) [inline frontend/src/components/Input.tsx:22]: bro this should be `boolean`, not `true`

**Review by zaxutic — COMMENTED** [review]
???

File: frontend/src/components/Input.tsx (hunk 4 above)
Comment (zaxutic) [reply frontend/src/components/Input.tsx:22]: oh fuck

**Pull request merged**